internal/http/services/owncloud/ocdav/versions.go

```diff
@@ -120,6 +130,7 @@ func (h *VersionsHandler) doListVersions(w http.ResponseWriter, r *http.Request,
 		return
 	}
 	if lvRes.Status.Code != rpc.Code_CODE_OK {
+		fmt.Printf("\n\n status %v \n\n", lvRes.Status)
```

Comment:
This crept in from debugging?

Reply:
Ah damn it...
Yes, forgot to remove it...